devdoc/umock_c_lib_requirements_with_ids.md

```diff
@@ -893,6 +893,11 @@ TEST_FUNCTION(validate_argument_as_type_sample)
 }
 ```
 
+### call_cannot_fail_func_{name}  
+
+XX**SRS_UMOCK_C_LIB_31_209: [** `call_cannot_fail_func__{name}` call modifier shall record that when performing failure case run, this call should be skipped. **]**  
```

Comment:
>_fail_ [](start = 44, length = 6)

Can you put a space between the 2 lines for consistency with the rest of the doc? Thanks!

Reply:
Could you be more specific on this one?  The way its rendering for me it looks like its following convention (the line 896 call_cannot_fail has two spaces at end of it, which isn't needed i guess, but line 897 I'm seeing a return and I'm not seeing spaces on other places in doc).

---
In reply to: [224577802](https://github.com/Azure/umock-c/pull/44#discussion_r224577802) [](ancestors = 224577802)

Reply:
Oh yes, my bad. leave it as is.

---
In reply to: [224880759](https://github.com/Azure/umock-c/pull/44#discussion_r224880759) [](ancestors = 224880759,224577802)